<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommentEditor</class>
 <widget class="QWidget" name="CommentEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#container
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.523136, x2:1, y2:0.528409, stop:0 rgba(81, 161, 249, 255), stop:0.352273 rgba(150, 108, 230, 255), stop:0.778409 rgba(220, 139, 151, 255), stop:1 rgba(234, 166, 73, 255));

	color: white;

	border-radius: 8;
}


#container #mod_date_lbl
{
	color: &quot;#310FDA&quot;;
}



#container #body_txte
{
	color:  white;
	background-color: transparent;

	outline: 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="container" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="text_layout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="post_date_lbl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>10</pointsize>
            <fontweight>DemiBold</fontweight>
           </font>
          </property>
          <property name="text">
           <string>Mon, 24 Sept  @username â–º 11:35 am</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mod_date_lbl">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Inter</family>
            <pointsize>8</pointsize>
            <fontweight>DemiBold</fontweight>
           </font>
          </property>
          <property name="text">
           <string>modified: 16/08/2024 11:25</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="body_txte">
        <property name="font">
         <font>
          <family>Inter</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="plainText">
         <string>a basic text to test the dimensions inside the box of at least three rows or maybe 4. This way is posssible to see the</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
